PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/ShiftLockController.rbxmx

```diff
@@ -95,12 +95,12 @@ local function initialize()
 	ShiftLockIcon.Position = UDim2.new(0, 12, 0, 2)
 	ShiftLockIcon.BackgroundTransparency = 1
 	ShiftLockIcon.Image = IsShiftLocked and SHIFT_LOCK_ON or SHIFT_LOCK_OFF
-	ShiftLockIcon.Visible = IsShiftLockMode
+	ShiftLockIcon.Visible = true
 	ShiftLockIcon.Parent = frame
 	
 	ShiftLockIcon.MouseButton1Click:connect(onShiftLockToggled)
 	
-	ScreenGui.Parent = PlayerGui
+	ScreenGui.Parent = IsShiftLockMode and PlayerGui or nil
```

Comment:
just to make sure I understand this, we are making screengui parenting the way we turn shiftlock icon on/off? why?